### PR TITLE
feat: Inkie landing

### PR DESCRIPTION
**Pull request by JeanBarriere**
**Missing in this PR**:
- [x] examples
  - [x] add missing services icon for examples
  - [x] check publish links are valid with studio

- [x] design
  - [x] check illustrations (may need to update some)
  - [x] update text sizes and color on some headers

- [x] add contact form to Netlify

### CODE REVIEW

**Comment by JeanBarriere** [comment]
@Arinono I guess you'll see this while I'm in the plane:

We're missing the service icons, and then checking that the links are the proper ones
Also, I don't have the pdf (and I guess we should not upload it on the repo, but on a cdn ?)

I updated the illustration but there may be some new changes on the header, as I see there is still changes on that.
Otherwise, everything should be fine here

**Comment by Arinono** [comment]
>check publish links are valid with studio

So far so good, we need to add the examples to the studio though. The PRs are almost ready for that.

>Also, I don't have the pdf (and I guess we should not upload it on the repo, but on a cdn ?)

I temporarily used now.sh to host it. Would be better to have our custom CDN, but I'm not sure we have enough time for that.

File: src/views/Home/RequestAccess.vue [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+          <s-button
+            state="primary"
+            :rounded="false"
+            borderless
+            full
+            :loading="sending"
+            :disabled="sending || success || !email"
+            type="submit"
+          >
+            {{ success ? 'Thanks !' : 'Submit' }}
```
Comment (Arinono) [inline src/views/Home/RequestAccess.vue]:
```suggestion
            {{ success ? 'Thanks!' : 'Submit' }}
```

🥖 🇫🇷

**Comment by JeanBarriere** [comment]
@Arinono thanks for the changes

**Pull request merged**